cache_test.go

```diff
@@ -23,3 +24,28 @@ func TestGetCache(t *testing.T) {
 		t.Errorf("want %q but got %q", want, got)
 	}
 }
+
+func TestGetCacheExpiration(t *testing.T) {
+	t.Parallel()
+	url := "https://golang.org"
+	cache := links.NewCache()
+	cache.SetTTL(1)
+	got, ok := cache.Get(url)
+	if ok {
+		t.Fatalf("cache should empty but %q was found", got)
+	}
+	want := "johniscool"
+	cache.Store(url, want)
+	got, ok = cache.Get(url)
+	if !ok {
+		t.Fatal("fail to retrieve stored value")
+	}
+	if want != got {
+		t.Errorf("want %q but got %q", want, got)
+	}
+	time.Sleep(1 * time.Second)
```

Comment:
```suggestion
	time.Sleep(time.Second)
```

Reply:
A second is a long time in tests; I suggest you reduce this to the smallest TTL that works reliably. A few milliseconds is probably enough.